R/core.R

```diff
@@ -40,8 +41,8 @@ qsearch <- function(str) {
 #' @examples
 #' qpeek("akarve/examples")
 #' qpeek("akarve/examples", TRUE)
-qpeek <- function(str, robust=FALSE) {
-    path <- paste0("~/quilt_packages/", str, ".json")
+qpeek <- function(str, robust=FALSE, package_path = "~/quilt_packages/") {
```

Comment:
If we're going to have a default search path, it should be set as an R option, so if a user has a non-standard directory, they don't need to specify it every time they use the command. This also will help reproducibility and code sharing.

See https://github.com/hadley/devtools/blob/master/R/zzz.r for an example of how to set up the options.